homeworks/nikolay.krasniuk_Kingstar22/9-animations/index.html

```diff
@@ -0,0 +1,214 @@
+                        <li class="header__item">
+                            <a class="header__item-link" href="#">How it Works</a>
+                        </li>
+                    </ul>
+                </nav>
+                <div data-header-burger class="header__burger">
+                    <span class="header__burger-line"></span>
+                </div>
+            </div>
+            <div class="header__btns">
+                <button data-btn type="button" class="header__btns-theme btn">
+                    <svg class = "header__btns-icon">
+                        <use data-icon xlink:href="#moon"></use>
+                    </svg>
+                </button>
+                <button type="button" class="header__btns-in btn">Sign In</button>
+                <button type="button" class="header__btns-up btn">Sign Up</button>
+            </div>
+        </header>
+        <section class="dream container">
+            <h1 class="dream__title">
+                Build Your Dream <span>House</span>
+            </h1>
+            <main class="dream__content">
+                <article class="dream__desc">
+                    <p class="dream__desc-text">The first and most trusted marketplace of design & build house in The World. 100% guaranteed Build safe, comfortable and transparent with a project management for the best result</p>
+                    <button type="button" class="dream__desc-btn btn">Free Consultation</button>
+                </article>
+                <picture class="dream__content-picture">
+                    <source srcset="img/img2_2x.png">
+                    <img class="dream__content-img img-animation" src="img/img2.png" alt="img">
+                </picture>
+            </main>
+        </section>
+        <section class="room container">
+            <h2 class="room__title">Build professional valuable room for you</h2>
+            <main class="room__gallery">
+                <picture class="room__gallery-picture">
+                    <source srcset="img/img-01_2x.png">
+                    <img class="room__gallery-img img-animation" src="img/img-01.png" alt="img">
+                </picture>
+                <picture class="room__gallery-picture">
+                    <source srcset="img/img-02_2x.png">
+                    <img class="room__gallery-img img-animation" src="img/img-02.png" alt="img">
+                </picture>
+                <picture class="room__gallery-picture">
+                    <source srcset="img/img-03_2x.png">
+                    <img class="room__gallery-img img-animation" src="img/img-03.png" alt="img">
+                </picture>
+                <picture class="room__gallery-picture">
+                    <source srcset="img/img-04_2x.png">
+                    <img class="room__gallery-img img-animation" src="img/img-04.png" alt="img">
+                </picture>
+            </main>
+        </section>
+        <section class="archmove container">
```

Comment:
`container` is a terrible name for the variable. So instead, you need to specify the purpose of this block.
I reviewed the code and can say it's better to use the `page__section` class name here. Or you can suggest a better name for it